plugins/azure-devops-backend/src/api/AzureDevOpsApi.ts

```diff
@@ -138,6 +142,103 @@ export class AzureDevOpsApi {
 
     return pullRequests;
   }
+
+  public async getBuildDefinitions(
+    projectName: string,
+    definitionName: string,
+  ): Promise<BuildDefinitionReference[]> {
+    this.logger?.debug(
+      `Calling Azure DevOps REST API, getting Build Definitions for ${definitionName} in Project ${projectName}`,
+    );
+
+    const client = await this.webApi.getBuildApi();
+    return client.getDefinitions(
+      projectName,
+      definitionName,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+    );
```

Comment:
Do you need to specify these optional parameters?
```suggestion
    return client.getDefinitions(projectName, definitionName);
```

Reply:
This works, I remember having issues with the other API calls so put them in assuming I needed them here too, thanks for pointing this out 👍